src/I18n/MessagesFileLoader.php

```diff
@@ -167,20 +171,12 @@ public function translationsFolders(): array
         if (!$localePaths && defined('APP')) {
```

Comment:
I wonder why the existence of `APP` is checked here when the constant used is `ROOT`. Seems like an oversight, would be a good idea to fix this too.